//abnormal condition occurs during program execution and 
//terminates the flow of a program

// exception handling is all about protecting our program from being terminated

//try catch finally throw throws

public class ExceptionDemo {

	public static void main(String[] args) {

		
		System.out.println("line 1");
		try {
		System.out.println(10/0);
		}
		catch (ArithmeticException e) {
			
			System.out.println("cannot divide a number by zero !! which school?");
		}
		
		System.out.println("line 3");
		
		
	}

}
